From 01a7433b06146ccf03c9cf71e9fbd55b3a24df99 Mon Sep 17 00:00:00 2001
From: nxa13443 <chaofan.huang@nxp.com>
Date: Thu, 29 Mar 2018 09:52:39 +0800
Subject: [PATCH 3547/5242] MLK-17936[IMX8QXP B0] Change repeated compatible
 name in vpu B0 encoder and decoder driver

commit  1718aa2c21696018f327ca914fe093fefbdf4057 from
https://source.codeaurora.org/external/imx/linux-imx.git

change a repeated line in vpu B0 encoder and decoder driver, missed qxp

Signed-off-by: nxa13443 <chaofan.huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/vpu-decoder-b0/vpu_b0.c         |    2 +-
 drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mxc/vpu-decoder-b0/vpu_b0.c b/drivers/mxc/vpu-decoder-b0/vpu_b0.c
index ce33722..0442065 100755
--- a/drivers/mxc/vpu-decoder-b0/vpu_b0.c
+++ b/drivers/mxc/vpu-decoder-b0/vpu_b0.c
@@ -2642,7 +2642,7 @@ static int vpu_resume(struct device *dev)
 
 static const struct of_device_id vpu_of_match[] = {
 	{ .compatible = "nxp,imx8qm-b0-vpudec", },
-	{ .compatible = "nxp,imx8qm-b0-vpudec", },
+	{ .compatible = "nxp,imx8qxp-b0-vpudec", },
 	{}
 }
 MODULE_DEVICE_TABLE(of, vpu_of_match);
diff --git a/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c b/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c
index fe3886d..8d8609f 100644
--- a/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c
+++ b/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c
@@ -2051,7 +2051,7 @@ static int vpu_resume(struct device *dev)
 
 static const struct of_device_id vpu_of_match[] = {
 	{ .compatible = "nxp,imx8qm-b0-vpuenc", },
-	{ .compatible = "nxp,imx8qm-b0-vpuenc", },
+	{ .compatible = "nxp,imx8qxp-b0-vpuenc", },
 	{}
 }
 MODULE_DEVICE_TABLE(of, vpu_of_match);
-- 
1.7.9.5

